server/src/main/java/org/elasticsearch/cluster/node/DiscoveryNodes.java

```diff
@@ -337,6 +337,7 @@ public DiscoveryNode resolveNode(String node) {
             return StreamSupport.stream(this.spliterator(), false).map(DiscoveryNode::getId).toArray(String[]::new);
         } else {
             ObjectHashSet<String> resolvedNodesIds = new ObjectHashSet<>(nodes.length);
+
```

Comment:
The changes to this file are now only whitespace, would you revert them please?

Reply:
Done.